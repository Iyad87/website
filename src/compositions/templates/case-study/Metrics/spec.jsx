import React from 'react';
import { shallow } from 'enzyme';
import Metrics from './index';

test('components/Logo snapshot', () => {
  const wrapper = shallow(
    <Metrics parameter="reviews" value="2.8" change="20%" />,
  );

  expect(wrapper).toMatchSnapshot();
});
