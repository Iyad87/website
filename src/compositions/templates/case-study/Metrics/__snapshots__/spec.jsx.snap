// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Logo snapshot 1`] = `
<React.Fragment>
  <PageSection
    gradient={false}
    gradientHeight="50%"
  >
    <Typography__H2>
      Metrics
    </Typography__H2>
    <Typography__base
      large={true}
    >
      some text here as well
    </Typography__base>
    <Typography__base
      newLines={true}
    >
      some text
    </Typography__base>
  </PageSection>
  <PageSection
    gradient={false}
    gradientHeight="50%"
  >
    <MetricGrid
      content={
        Array [
          Object {
            "change": "20%",
            "parameter": "test",
            "value": "20.00",
          },
        ]
      }
    />
  </PageSection>
</React.Fragment>
`;
